tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -637,8 +625,8 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
                 return nil, sharedSet, err
             }
         } else {
-            whisk.Debug(whisk.DbgError, "--kind argument '%s' is invalid\n", flags.action.kind)
-            errMsg := fmt.Sprintf("'%s' is not a supported action runtime", flags.action.kind)
+            whisk.Debug(whisk.DbgError, "Action runtime exention '%s' is not supported\n", ext)
```

Comment:
I'm not sure we're consistent wrt supported file extensions.  If the --kind flag has a valid kind value, it seems that the extension is not checked, so the file could have any extension or none if the --kind is either swift:3 or nodes:6